src/window_identifier/mod.rs

```diff
@@ -310,6 +315,109 @@ impl Serialize for WindowIdentifierType {
     }
 }
 
+impl WindowIdentifierType {
+    /// Sets the given window as a modal child of the window represented by
+    /// self. Currently, setting Wayland window as parent of Wayland window and
+    /// X11 window as parent of X11 window is supported.
```

Comment:
Can you instead write this as a list of cases and mention next to each one whether it is a supported one or not?

Reply:
Done.